lib/travis/api/v3/routes/dsl.rb

```diff
@@ -34,7 +34,7 @@ def with_resource(resource)
     end
 
     def route(value)
-      current_resource.route = prefix + value
+      current_resource.route = Mustermann.new(prefix) + Mustermann.new(value)
```

Comment:
This is to avoid the second part to get escaped. ie, `Mustermann.new('/:name') + '/:id'` results in `Mustermann.new("/:name/\\:id")` (second capture gets escaped).